The `tatsu-lts` dependency is apparently interchangeable with mainline `tatsu`,
except that it maintains compatibility with old versions of Python (see its PyPI 
page). Since conda-forge is always keeping up with the latest Python versions, 
it doesn't make sense for us to package both, and we can just drop in mainline 
`tatsu` instead.

diff --git a/pyproject.toml b/pyproject.toml
index 59c7c69..0fe5fda 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -35,7 +35,7 @@ dependencies = [
     'beancount >= 2.3.4',
     'click > 7.0',
     'python-dateutil >= 2.6.0',
-    'tatsu-lts',
+    'tatsu',
 ]
 
 [project.optional-dependencies]
